appendix-basic10.tex

```diff
@@ -1960,8 +1960,10 @@ \subsection{DMA}
 \begin{tcolorbox}[colback=black,coltext=white]
 \verbatimfont{\codefont}
 \begin{verbatim}
-DMA 3, 2000,   32,0,  2048,0 :REM FILL SCREEN WITH BLANKS
-DMA 0, 2000, 2048,0, 16384,1 :REM COPY SCREEN TO $14000
+DMA 0, 80*25, 2048, 0, 0, 1  :REM SAVE SCREEN TO $00000 BANK 1
```

Comment:
Provides more interesting examples of dma command use.

Reply:
Please change the target to BANK 4 instead of BANK 1, as BANK 1 at that location contains DOS variables. It will make the example MEGA65-specific, but that's fine.

Reply:
Please change "MEGA 65" to "MEGA65" (how the name of the computer should always be written).

Reply:
> Please change "MEGA 65" to "MEGA65" (how the name of the computer should always be written).

9c39cf63961c23c9dcfa7a49e0d861216a2bb0e7

Reply:
> Please change the target to BANK 4 instead of BANK 1

9a9b646f36fa7629a5288cd966b2d76a20a758d1